### PR TITLE
Release 1.2.5

### PR DESCRIPTION
**Pull request by bredelings**
Changes for releasing v1.2.5
- Update NEWS.md
- Change version to 1.2.5
- Remove dnInverse
- Update scripts for generating executables
- Update scripts for singularity build

### CODE REVIEW

**Review by bjoelle — APPROVED** [review]
looks fine to me, thanks for writing out all the changes !

File: NEWS.md [hunk 1]
```diff
@@ -1,46 +1,63 @@
...
+  * Types
+      - Fix type conversion to integer so that it employs deterministic nodes (#545).
+      - Make `RealPos` coherent between conversion and construction (#554).
+      - Make `vectorFlatten` work on more type (#514).
+  * Crash / NaN
+      - Fix `dnMixture` of rate matrices and other `Cloneable` objects (#501).
+      - Fix `treeAssembly` sometimes failing to initialize branch lengths (#509).
+      - Fix crash if a file changes while we are `source( )`-ing it (#510).
+      - Fix segfault with `dnConstrainedTopology` + `dnUniformTimeTree` (#513).
+      - Fix a segfault in Model::getOrderedStochasticNodes (#569).
```
Comment (davidcerny) [inline NEWS.md:30]: Change Model::getOrderedStochasticNodes -> `Model::getOrderedStochasticNodes`

File: NEWS.md [hunk 2]
```diff
@@ -1,46 +1,63 @@
...
+  * MCMC: initialization
+      - Fix fossil age sampling with an initial tree in `dnBDSTP` (#480).
+      - Fix FBD initialization issues (#537, #561).
+      - Extend the starting tree simulator to account for origin/root age (#561).
+  * MCMC
+      - Fix recalculation of likelihoods for brownian motion distributions (#596).
```
Comment (davidcerny) [inline NEWS.md:36]: Capitalize "Brownian"

File: NEWS.md [hunk 3]
```diff
@@ -1,46 +1,63 @@
...
+      - Fix recalculation of likelihoods with mvRootNodeTimeSlideUniform (#594).
```
Comment (davidcerny) [inline NEWS.md:37]: Change mvRootNodeTimeSlideUniform -> `mvRootTimeSlideUniform`

File: NEWS.md [hunk 4]
```diff
@@ -1,46 +1,63 @@
...
+      - Fix recalculation of likelihood with treeAssembly (#549).
```
Comment (davidcerny) [inline NEWS.md:38]: Change treeAssembly -> `treeAssembly`

File: NEWS.md [hunk 5]
```diff
@@ -1,46 +1,63 @@
...
+      - Fix Gelman-Rubin (PSRF) and stationarity stopping rules (#555).
+      - Don't move fossil tips outside their age ranges (#559).
+      - Fix operator summary for MC^3 when moves are tuned (#522).
+  * Debug: Log reason for -Inf and NaN probabilities.
```
Comment (davidcerny) [inline NEWS.md:42]: Change this to:

* Debug
    - Log reason for -Inf and NaN probabilities ([#592](https://github.com/revbayes/revbayes/pull/592)).
    
Arguably, PRs [#570](https://github.com/revbayes/revbayes/pull/570), [#575](https://github.com/revbayes/revbayes/pull/575), and [#593](https://github.com/revbayes/revbayes/pull/593) should also be listed under this subheading.

File: NEWS.md [hunk 6]
```diff
@@ -1,46 +1,63 @@
...
+  * Misc
+      - Allow reading non-square matrices (#564).
+      - Allow checking `args.size()` when no arguments are given. (#479).
+      - Balance braces when printing Matrix<Real> (615).
```
Comment (davidcerny) [inline NEWS.md:46]: Link to PR #615.

File: NEWS.md [hunk 7]
```diff
@@ -1,46 +1,63 @@
...
-  * The `weight` parameter of moves and `moveschedule` parameter of `mcmc` (#506).
+  * `mcmc` and `mcmcmc`
+    - `.run( )` (#485, #488).
+    - `.initializeFromCheckpoint( )` (#505).
+    - `moveschedule` parameter and the `weight` parameter of moves (#506).
   * Corrections to `dnBivariatePoisson` (#539) and `mcmcmc` (#541).
-  
-## Changes
-  * Refactor `dnExp` and `dnLnorm` to consolidate them with their offset versions (#503).
-  * Add invariant checks to the likelihood calculator in `AbstractPhyloCTMCSiteHomogeneous` (#550).
```
Comment (davidcerny) [inline NEWS.md]: I guess these are no longer listed, because they are not user-facing?

File: NEWS.md [hunk 8]
```diff
@@ -1,46 +1,63 @@
-# RevBayes 1.2.5 (unreleased)
+# RevBayes 1.2.5 (Dec 19, 2025)
+
+## Backwards-incompatible changes
+  * Remove `underPrior` argument to `mcmc.run( )` and `mcmcmc.run( )`.  You can use `model.ignoreAllData()` instead.
+  * Remove `dnLogexponential`. You can use `dnExponential(l) |> tnLog()` instead.
+  * Rename `tree.makeBifurcating()` to `tree.suppressOutdegreeOneNodes()`.
```
Comment (davidcerny) [inline NEWS.md:6]: I would phrase it as "Remove `tree.makeBifurcating()` in favor of `tree.suppressOutdegreeOneNodes()`". The method wasn't just renamed; its behavior was changed, too.

**Review by davidcerny — CHANGES_REQUESTED** [review]
Looks good overall! I understand the list is long enough as it is, but PRs [#605](https://github.com/revbayes/revbayes/pull/605) and [#613](https://github.com/revbayes/revbayes/pull/613) might also warrant a mention.

File: NEWS.md (hunk 1 above)
Comment (bredelings) [reply NEWS.md:30]: Done.

File: NEWS.md (hunk 2 above)
Comment (bredelings) [reply NEWS.md:36]: OK.

File: NEWS.md (hunk 3 above)
Comment (bredelings) [reply NEWS.md:37]: Fixed.

File: NEWS.md (hunk 5 above)
Comment (bredelings) [reply NEWS.md:42]: I was debating that...  sure,  I'll add those.

File: NEWS.md (hunk 7 above)
Comment (bredelings) [reply NEWS.md]: Yeah.

File: NEWS.md (hunk 8 above)
Comment (bredelings) [reply NEWS.md:6]: Ah, I see.

File: NEWS.md (hunk 6 above)
Comment (bredelings) [reply NEWS.md:50]: Fixed.

**Comment by bredelings** [comment]
> Looks good overall! I understand the list is long enough as it is, but PRs https://github.com/revbayes/revbayes/pull/605 and https://github.com/revbayes/revbayes/pull/613 might also warrant a mention.

I added #605.  I think #613 will become user-facing in the next release, so we can mention it when that happens.

**Review by davidcerny — APPROVED** [review]

**Pull request merged**